compliance-report/generate.sh

```diff
@@ -2,6 +2,23 @@
 
 set -eux
 
+# Get release creator.
+cd $GITHUB_WORKSPACE
+export GH_TOKEN=${GH_TOKEN:-TOKEN}
+RELEASE_CREATOR=$(gh api users/${GITHUB_ACTOR} --jq '.name')
+
+# Handle security report.
+SECURITY_REPORT="N/A"
+if [ -n "$SECURITY_REPORT_LOCATION" ]; then
+  if [[ $SECURITY_REPORT_LOCATION == https* ]]; then
+    SECURITY_REPORT="See $SECURITY_REPORT_LOCATION"
+  else
+    SECURITY_REPORT="See https://github.com/$GITHUB_REPOSITORY/blob/$RELEASE_VERSION/$SECURITY_REPORT_LOCATION"
+  fi
+elif [ -n "$SECURITY_REPORT_URL" ]; then
+    SECURITY_REPORT="See $SECURITY_REPORT_URL"
```

Comment:
I didn't notice at first how the Python script injected this, until I realised that this variable is read from the AWS Secrets Vault.

IMO, the documentation for this action should explain the possibilities of specifying the security report:
* By specifying an absolute URL starting with `https`,
* By specifying a relative path, which is then linked to the corresponding git blob for the tagged version
* By adding the `security-report-url` to the AWS Secrets Vault

Reply:
Updated